src/common/format.js

```diff
@@ -23,12 +23,16 @@ export function formatNumber(number) {
   return formatThousands(number);
 }
 
+export function formatDecimal(number, precision = 2) {
+  return (Math.round((number || 0) * 100) / 100).toFixed(precision);
+}
```

Comment:
How is this meaningfully distinct from just doing `number.toFixed(precision);` ?

Reply:
It isn't, removed it.